keyboards/Foostan-CRKBD-blok-L/main.py

```diff
@@ -8,6 +8,8 @@
 from kmk.extensions.peg_oled_Display import Oled,OledDisplayMode,OledReactionType,OledData
 from kmk.extensions.peg_rgb_matrix import Rgb_matrix
 from kmk.modules.split import Split, SplitSide, SplitType
+from kmk.extensions.media_keys import MediaKeys
+keyboard.extensions.append(MediaKeys())
 keyboard = KMKKeyboard()
```

Comment:
If I'm not mistaken, the `keyboard` object here is being used on line 12 prior to invocation of `KMKKeyboard()` on line 13.

This will probably lead to issues.

Reply:
Yep, I think you're right. Thanks.